packages/v4/patternfly-docs/content/design-guidelines/components/toggle-group/toggle-group.md

```diff
@@ -10,39 +10,35 @@ Toggle items can consist of icons, text, or both. Styling of toggles is dependen
 
 <img src="./img/elements-items.png" alt="Toggle group elements" width="1536"/>
 
-## Toggle group types
+## Usage
+Toggle groups can be used when the user must select a single choice from a mutually exclusive set of options and can be used with or without icons. 
+Use cases also vary depending on variation. 
+
+## Variations
 
 1. [Single select toggle group](#single-select-toggle-group): Use to toggle between multiple items
 2. [Multi select toggle group](#multi-select-toggle-group): Use to select multiple attributes
 3. [Toggle items](#toggle-items): Use to select a single action
 
 ### Single select toggle group
-
-#### Usage
 Use single select toggle groups when you want to toggle between multiple items. There are many use cases where this can apply, such as:
-  - Selecting a single action within a group of actions 
+  - Selecting a single action within a group of actions (this can be useful for topology or similar canvas views).
```

Comment:
This bullet has a period at the end, but the other two don't - pick one and go with it!